pkg/daemon/writer.go

```diff
@@ -103,9 +103,9 @@ func (nw *NodeWriter) SetUpdateDegraded(err error, client corev1.NodeInterface,
 // node to degraded due to that error.
 func (nw *NodeWriter) SetUpdateDegradedIgnoreErr(err error, client corev1.NodeInterface, node string) error {
 	// log error here since the caller won't look at it
-	degraded_err := nw.SetUpdateDegraded(err, client, node)
-	if degraded_err != nil {
-		glog.Errorf("Error while setting degraded: %v", degraded_err)
+	degradedErr := nw.SetUpdateDegraded(err, client, node)
```

Comment:
To keep it consistent we should probably just make it `if degradedErr := nw.SetUpdateDegraded(err, client, node); degradedErr != nil` here right?